src/module.c

```diff
@@ -126,7 +126,7 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
 	RedisModule_Log(ctx, "notice", "Thread pool created, using %d threads.", threadCount);
 
 	// Initialize array of command contexts
-	command_ctxs = calloc(threadCount + 1, sizeof(CommandCtx*));
+	command_ctxs = calloc(threadCount + 1, sizeof(CommandCtx *));
```

Comment:
Why the extra space in `sizeof(CommandCtx *));`
I prefer it to be `sizeof(CommandCtx*));`